directory_sso_api_client/user.py

```diff
@@ -4,8 +4,9 @@
 class UserAPIClient(BaseAPIClient):
 
     endpoints = {
-        'session_user': 'session-user/',
-        'last_login': 'last-login/',
+        'session_user': 'api/v1/session-user/',
```

Comment:
please change the SSO client base env var in each app that uses that in dev env + make a PR for stage / prod conf